presto-docs/src/main/sphinx/release/release-0.249.rst

```diff
@@ -0,0 +1,38 @@
+=============
+Release 0.249
+=============
+
+**Highlights**
+==============
+* Add Support for Hive Connector to work with Hive 3 metastore for basic tables.
+* Update joda to 2.10.8 tzdata 2020d.
+* Support computing CRC32 checksum for data exchanges.
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix an error executing index joins when join spilling is enabled.
+* Enforce ``max_total_memory_per_node`` on user memory allocation.
+* Update joda to 2.10.8 tzdata 2020d. Make sure you deploy with a JVM that is using matching tzdata.
```

Comment:
I think joda should be capitalized. And in the documentation it's always referred to as Joda-Time, so I guess we should refer to it that way too. (here and in the highlight)